kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -68,6 +68,8 @@
   public static final String KUBERNETES_ALL_PROXY = "all.proxy";
   public static final String KUBERNETES_NO_PROXY = "no.proxy";
 
+  public static final String KUBERNETES_USER_AGENT = "fabric8-kubernetes-client";
```

Comment:
@jimmidyson: We could have a filtered resource under src/main/resources that would contains ${project.version} this would be substituted during build and could be loaded by Config in a static block.

T